engines/pytorch/pytorch-engine/src/main/java/ai/djl/pytorch/engine/PtNDArrayEx.java

```diff
@@ -694,7 +696,59 @@ public NDList multiBoxPrior(
             List<Float> steps,
             List<Float> offsets,
             boolean clip) {
-        throw new UnsupportedOperationException("Not implemented");
+        //Based on https://github.com/apache/mxnet/blob/a720b15b5fa011a9610dfaeabf0792443c6abec5/src/operator/contrib/multibox_prior.cc
+
+        NDManager ndManager = array.getManager().getParentManager();
+
+        Float step_x   = steps.get(1);
```

Comment:
Please follow our coding convention, we use google AOSP format

Reply:
Could you help me out here? It's a big document. Are you referring to a particular rule?